Formula/xfig.rb

```diff
@@ -0,0 +1,76 @@
+class Xfig < Formula
+  desc "Facility for interactive generation of figures"
+  homepage "https://mcj.sourceforge.io"
+  url "https://downloads.sourceforge.net/mcj/xfig-3.2.8a.tar.xz"
+  sha256 "ba43c0ea85b230d3efa5a951a3239e206d0b033d044c590a56208f875f888578"
+  license "MIT"
+
+  livecheck do
+    url "https://downloads.sourceforge.net/mcj/"
+    regex(/href=.*?xfig[._-](\d+(?:\.\d+)+[a-z]?)\.t/i)
```

Comment:
```suggestion
    url "https://sourceforge.net/projects/mcj/files/"
    regex(/href=.*?xfig[._-](\d+(?:\.\d+)+[a-z]?)\.t/i)
    strategy :page_match
```

As it currently is, this `livecheck` block does not work because the `Sourceforge` strategy checks the RSS feed for the project, where I don't think `href`s are present. Furthermore, there are archives other than `xfig`, which is the one we're interested in. In such cases it is preferable to force `livecheck` to use `PageMatch` on the list of files instead. I've added a suggestion that does this, and reports the latest version as `3.2.8a`.

Reply:
Thanks for the fix. For reference, the sourceforge feed from the overview page, https://sourceforge.net/p/mcj/activity/feed, contains all news from this project, and also
```xml
<item><title>Xfig released /xfig-3.2.8a.tar.xz</title>
<link>https://sourceforge.nethttps%3A//sourceforge.net/projects/mcj/files/xfig-3.2.8a.tar.xz/download</link>
<description></description>
<dc:creator xmlns:dc="http://purl.org/dc/elements/1.1/">Xfig</dc:creator>
<pubDate>Sat, 27 Mar 2021 19:37:03 -0000</pubDate>
<guid isPermaLink="false">https://sourceforge.net/projects/mcj/files/xfig-3.2.8a.tar.xz/download</guid></item>
```
There is a different feed on the files page, https://sourceforge.net/projects/mcj/rss?path=/., which seems more structured and contains information on all files and past submissions,
```xml
    <item>
      <title><![CDATA[/xfig-3.2.8a.tar.xz]]></title>
      <link>https://sourceforge.net/projects/mcj/files/xfig-3.2.8a.tar.xz/download</link>
      <guid>https://sourceforge.net/projects/mcj/files/xfig-3.2.8a.tar.xz/download</guid>
      <pubDate>Sat, 27 Mar 2021 19:36:13 UT</pubDate>
      <files:sf-file-id xmlns:files="https://sourceforge.net/api/files.rdf#">35391842</files:sf-file-id>
      <files:extra-info xmlns:files="https://sourceforge.net/api/files.rdf#">tar archive</files:extra-info>
      <media:content xmlns:media="http://video.search.yahoo.com/mrss/" type="application/x-xz; charset=binary" url="https://sourceforge.net/projects/mcj/files/xfig-3.2.8a.tar.xz/download" filesize="5380896"><media:hash algo="md5">58dd2b6f9f17d7006c78156ce2da0073</media:hash></media:content>
    </item>
```